### ISSUE TITLE
Apps: keep page location when changing compute parameters

### ISSUE THREAD

**Issue by asizemore** [VEuPathDB/web-eda#1545]
On the mbio beta site, if we change an app parameter it jumps us to the top of the page. Can we modify these selections so that we keep the page location, similar to changing a plot variable?

Also observed in the new compute-service-integration branch.

**Comment by jernestmyers** [comment]
I'm not observing the undesirable scroll-jump behavior anymore when changing parameters. 

While investigating the issue, I spun up the latest code on a `web-eda` dev server, and the scroll-jump didn't occur when changing parameters. Then, I checked on the feature mbio site, and the scroll-jump also didn't occur when changing parameters. I tested on both Chrome and Firefox (with MacOS Ventura v13).

Perhaps this was solved in some other ticket? 🤷
